python/lsst/meas/base/forcedMeasurement.py

```diff
@@ -335,26 +325,18 @@ def run(self, exposure, refCat, refWcs, idFactory=None, exposureId=None, beginOr
                 endOrder = endOrder,
             )
 
-        return lsst.pipe.base.Struct(sources=sources)
-
     def generateSources(self, exposure, refCat, refWcs, idFactory=None):
-        """!
-        Initialize an output SourceCatalog using information from the reference catalog.
+        """!Initialize an output SourceCatalog using information from the reference catalog.
 
-        This generate a new blank SourceRecord for each record in refCat, copying any
-        fields in ForcedMeasurementConfig.copyColumns.  This also transforms the Footprints
-        in refCat to the measurement coordinate system if it differs from refWcs, and attaches
-        these to the new SourceRecords.  Note that we do not currently have the ability to
-        transform heavy footprints, so when the reference and measure WCSs are different,
-        HeavyFootprints will be converted to regular Footprints, which makes it impossible
-        to properly measure blended objects.
+        This generates a new blank SourceRecord for each record in refCat.  Note that footprints
+        are attached to the sources using the class's implementation of attachFootprints().
```

Comment:
This last sentence now seems to be inaccurate: better to say that this method does *not* attach any `Footprints`, and doing so is up to the caller (who can call `attachedTransformedFootprints` if they don't have anything better).